test/page-objects/searchItemPage.js

```diff
@@ -1,24 +1,24 @@
-import { Label } from "../../framework/elements/index.js"
-import BasePage from "../../framework/page/BasePage.js"
+import { Label } from "../../framework/elements/index.js";
+import BasePage from "../../framework/page/BasePage.js";
 
-class searchItemPage extends BasePage {
+class SearchItemPage extends BasePage {
   constructor() {
-    super(new Label("//h1[@id='largeiteminfo_item_name']", "Item Label"), "Item Label")
+    super(new Label("//h1[@id='largeiteminfo_item_name']", "Item Label"), "Item Label");
 
-    this.itemGameName = new Label("//div[@id='largeiteminfo_game_name']")
-    this.itemHero = new Label("//div[@class='descriptor' and contains(., 'Used By')]")
-    this.itemType = new Label("//div[@id='largeiteminfo_item_type']")
+    this.itemGameName = new Label("//div[@id='largeiteminfo_game_name']");
+    this.itemHero = new Label("//div[@class='descriptor' and contains(., 'Used By')]");
+    this.itemType = new Label("//div[@id='largeiteminfo_item_type']");
   }
 
-  async isItemInfoCorrect(gameName, heroName, rarityType) {
-    const itemGame = await this.itemGameName.getText()
-    const itemHero = await this.itemHero.getText()
-    const itemType = await this.itemType.getText()
+  async getItemInfo() {
+    const itemGame = await this.itemGameName.getText();
+    const itemHero = await this.itemHero.getText();
+    const itemType = await this.itemType.getText();
 
-    if (!gameName === itemGame || !itemHero.includes(heroName) || !itemType.includes(rarityType)) return false
+    console.log([itemGame, itemHero, itemType]);
 
-    return true
+    return [itemGame, itemHero, itemType];
   }
```

Comment:
you are tied to the order of values, and count of values. It could cause an issues in future

one text - one assertion
OR put it in dictionary (object with key-value pairs)

Reply:
one text - one assertion is more preferable, because sometimes you will need to get only one text from page, not all three